hinted-compilation/hinted-mods.py

```diff
@@ -86,7 +86,7 @@ def get_checklist(full_name):
     """
     if not full_name:  # guard against nonsense
         return []
-    mtab = full_name.split(".")  # separate components by dots
+    mtab = str(full_name).split(".")  # separate components by dots
```

Comment:
I think you want to use `splitPackageName` in a loop here, until the package name becomes `None`, that avoids the string operation.

Reply:
Must've missed that when reading the docs, I'll fix that.